tycho-bundles/org.eclipse.tycho.p2.resolver.impl.test/src/test/java/org/eclipse/tycho/p2/target/TestResolverFactory.java

```diff
@@ -47,6 +59,39 @@ public TestResolverFactory(MavenLogger logger) {
 
         targetDefinitionResolverService = new TargetDefinitionResolverService();
         targetDefinitionResolverService.setMavenContext(mavenContext);
+        targetDefinitionResolverService.setMavenDependenciesResolver(new MavenDependenciesResolver() {
+
+            @Override
+            public Collection<?> resolve(String groupId, String artifactId, String version, String packaging,
+                    String classifier, String dependencyScope, int depth,
+                    Collection<MavenArtifactRepositoryReference> additionalRepositories, Object session)
+                    throws DependencyResolutionException {
+                // This is supposed to mimic Maven repo returning an artifact
+                String relativePath = groupId.replace('.', File.separatorChar) + File.separator + artifactId
```

Comment:
Can we use GAV(....) here as it already has a method to construct a maven repo path?

Reply:
The only GAV accessible here doesn't have method to build a path as far as I could see. Or did I miss something>